src/lib.rs

```diff
@@ -110,6 +111,10 @@ pub enum DaemonizeError {
     __Nonexhaustive,
 }
 
+fn errno() -> Errno {
+    io::Error::last_os_error().raw_os_error().unwrap()
```

Comment:
Can we have `expect("last_os_error produces only OS errors hence raw_os_error cannot be None")` instead of unwrap?